inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -372,6 +376,14 @@ void GNADeviceHelper::open(uint8_t n_threads) {
 }
 
 void GNADeviceHelper::close() {
+    auto requestsToClose = unwaitedRequestIds;
```

Comment:
Why do we need this additional variable?

Reply:
unwaitedRequestsIds is modified during for loop restroying the validity of iterator